requirements.txt

```diff
@@ -1,5 +1,10 @@
 # These are the base Weave requirements, enough for weave tracking and evaluation
 # to work.
+# -r requirements.legacy.txt
+# COPIED FROM LEGACY:
```

Comment:
why not put this comment next to where you added these reqs?

Reply:
This is a temporary "inlining" of `-r requirements.legacy.txt` because the `-r ...` syntax is not valid for pip installs using pyproject.toml.  I think this form is more obvious that it's a copy-paste vs. annotating individual reqs

In future, we'll remove this altogether and just use pyproject.toml